maintainers/README.md

```diff
@@ -23,7 +23,7 @@ for the maintainers to provide feedback.
 
 For critical packages, this convention needs to be negotiated with the
 maintainer. A critical package is one that causes mass-rebuild, or where an
-author is listed in the [`CODEOWNERS`](../.github/CODEOWNERS) file.
+author is listed in the [`OWNERS`](../ci/OWNERS) file.
```

Comment:
I feel like this whole paragraph has been wrong for a while. `OWNERS` is for things that don't have a `meta.maintainers` as well, that may or may not be critical.

Reply:
But I don't want this to be a blocking comment. We can rewrite it in another PR. Everything else looks good to me.